pkgs/tools/networking/mpssh/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, openssh }:
+
+stdenv.mkDerivation rec {
+  pname = "mpssh";
+  version = "1.3.3";
+
+  src = fetchFromGitHub {
+    sha256 = "0iibzb9qrwf9wsmp2b8j76qw66ql8fw5jx02x82a22fsaznb5dng";
+    rev = "${version}";
+    repo = pname;
+    owner = "ndenev";
+  };
+
+  buildInputs = [ openssh ];
+
+  postPatch = ''
+    substituteInPlace Makefile --replace /usr/local $out
+    substituteInPlace Makefile --replace "-o root" ""
```

Comment:
Perhaps use a patch, it will make it easier to notice when the substitution no longer applies

```nix
patches = [
  # Do not install as root
  (fetchpatch {
    url = https://github.com/ndenev/mpssh/commit/3cbb868b6fdf8dff9ab86868510c0455ad1ec1b3.patch;
    sha256 = "0000000000000000000000000000000000000000000000000000000000000000";
  })
];
```